ledger/ledger-api-integration-tests/src/test/lib/scala/com/digitalasset/platform/apitesting/LedgerFactories.scala

```diff
@@ -56,6 +56,19 @@ object LedgerFactories {
 
   }
 
+  def createRemoteApiProxyResource(config: PlatformApplications.Config)(
+      implicit esf: ExecutionSequencerFactory): Resource[LedgerContext.SingleChannelContext] = {
+    require(config.remoteApiEndpoint.isDefined, "config.remoteApiEndpoint has to be set")
```

Comment:
you could assert on the `host` and `port` too

Reply:
`host` can be empty and `port` can be negative. It's nit-picking, I just thought I mention it.